### ISSUE TITLE
Question about the released pretrained model

### ISSUE THREAD

**Issue by ToBeCodeCreater** [kakaobrain/flame#5]
Thanks for your nice work! I downloaded the pretrained models and clip models, and tested them on HumanML3D and Babel datasets. Then, I got a really poor result on both datasets, as shown in following tabels. 
I am sure that the models I used for testing  were the released models.  Have you tested the performance of the released models?

|  HumanML3D | mCLIP | R-Precision(top 1) | FID |
| -- | -- | -- | -- |
paper | 0.297 | 0.513 | 21.15 
released | 0.1458 | 0.621 | 1478 

|  Babel | mCLIP | R-Precision(top 1) | FID |
| -- | -- | -- | -- |
paper |  0.318 | 0.888 | 53.003 
released | 0.1813 | 0.032 | 273.26

**Comment by ToBeCodeCreater** [comment]
And I trained two models using the released code on both datasets. I got results similar with the results reported in your paper on mCLIP, FID and R-Precision metrics. But the MID scores were negative numbers.  And I computed MID score using [your code](https://github.com/kakaobrain/flame/blob/bb93249a3fc3605d51c892350f7f5fd388cf3bb2/src/utils/eval_util.py#L222) and [the official code of MID](https://github.com/naver-ai/mid.metric/blob/08138654d45bedebd054de76c5e23af0efec8776/metrics/mid.py).

| | MID (reported in the paper) | MID ( the trained model) |
| --| -- | -- |
HumanML3D | 29.94 | -28.3 |
Babel | 53.003 | -4.7 |

I'm confused about this result. Could you tell me how to compute MID score correctly?

**Comment by jihoonerd** [comment]
Thanks for pointing out the issue.
I re-run the released code and found that the test process was not completed properly due to a bug.
This will be fixed in #8 .
With this, the sample generation in test stage should be completed properly, which yields a similar benchmark number reported in paper.
